tinymce-per-block/src/assets/stylesheets/main.scss

```diff
@@ -1,3 +1,4 @@
+@import '../../../../shared/index';
```

Comment:
I preferred including this here instead of `index.html` to keep the `npm run dev` command working